### PR TITLE
Feature/add init

### LINKED ISSUE #2: Add version flag
The tool should be able to display its version by passing the --version flag.

### PR DESCRIPTION
**Pull request by SheepFromHeaven**
resolves #2 and #5

### CODE REVIEW

File: src/actions/init.ts [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+const defaultProjectName = 'biotope-boilerplate';
+export const registerInit = commander => commander
+    .command('init [name]')
+    .action((name = defaultProjectName) => {
+        console.log(chalk.green('========================================'));
+        console.log(chalk.green('======== Pulling repository 🏎️ ========='));
+        console.log(chalk.green('========================================'));
+        gitClone(
+            'https://github.com/biotope/biotope-boilerplate.git',
+            name,
```
Comment (jurekbarth) [inline src/actions/init.ts]: Is it possible to make `name` optional and add a default name?

File: src/actions/init.ts [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+            () => {
+                process.chdir(name);
+                rimraf('./.git', () => {
+                    console.log(chalk.green('========================================'));
+                    console.log(chalk.green('====== Installing dependencies 💻 ======'));
+                    console.log(chalk.green('========================================'));
+                    npm.load((err) => {
+                        npm.commands.install((er, data) => {
+                            console.log(chalk.green('========================================'));
+                            console.log(chalk.green('====== Everything ready to go 👍 ======='));
```
Comment (jurekbarth) [inline src/actions/init.ts]: Nit: Remove the last `=` 🙃

File: src/actions/init.ts [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+                            console.log(chalk.green('========================================'));
+                        });
+                        npm.on('log', (message) => {
+                            console.log(message);
```
Comment (jurekbarth) [inline src/actions/init.ts]: Could you refactor with async/await, so we get rid of this callback chain?

**Review by jurekbarth — CHANGES_REQUESTED** [review]
Thanks for removing `lib`

File: src/actions/init.ts (hunk 1 above)
Comment (SheepFromHeaven) [reply src/actions/init.ts]: name IS optional. Default name is 'biotope-boilerplate'

Comment (jurekbarth) [reply src/actions/init.ts]: perfect 👍

**Review by jurekbarth — APPROVED** [review]

**Pull request merged**